.github/workflows/update-from-protocol-contracts-solana.yaml

```
@@ -0,0 +1,45 @@
+          rm -f src/pages/developers/protocol/solana.md
+          cp protocol-contracts-solana/docs/gateway.md src/pages/developers/protocol/solana.md
+          rm -rf protocol-contracts-solana
+
+      - name: Check for changes
+        id: check-changes
+        run: |
+          git add src/pages/developers/protocol
+          git diff --cached --exit-code || echo "::set-output name=changed::true"
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v3
```

actionlint (1.7.7)

the runner of "peter-evans/create-pull-request@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)